ci/infra/aws/master-instance.tf

```diff
@@ -1,27 +1,58 @@
-resource "aws_instance" "control_plane" {
-  ami                         = data.susepubliccloud_image_ids.sles15sp2_chost_byos.ids[0]
+resource "aws_security_group" "master" {
+  description = "security rules for master nodes"
+  name        = "${var.stack_name}-master"
+  vpc_id      = aws_vpc.platform.id
+
+  tags = merge(
+    local.tags,
+    {
+      "Name"  = "${var.stack_name}-master"
+      "Class" = "SecurityGroup"
+    },
+  )
+
+  # etcd - internal
+  ingress {
+    from_port   = 2379
+    to_port     = 2380
+    protocol    = "tcp"
+    cidr_blocks = [var.cidr_block]
+    description = "etcd"
+  }
+
+  # api-server - everywhere
```

Comment:
Same here, Api server should be internal only as well.

Reply:
`var.cidr_block` is internal.